src/services/authentication/src/validators/signupValidator.js

```diff
@@ -0,0 +1,56 @@
+const joi = require("joi");
+
+const userSchema = joi.object({
+  username: joi
+    .string()
+    .min(1)
+    .max(40)
+    .messages({
```

Comment:
These messages can be written by default, you needn't to explicitly write them like this.